plugin/src/main/java/org/opensearch/ml/action/profile/MLProfileAction.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.action.profile;
+
+import org.opensearch.action.ActionType;
+import org.opensearch.ml.constant.CommonValue;
+
+public class MLProfileAction extends ActionType<MLProfileResponse> {
+    // Internal Action which is not used for public facing RestAPIs.
```

Comment:
Minor, seems `MLProfileAction.INSTANCE` is used in `RestMLProfileAction.java`, remove this comment?